docscan/session/create/sdk_config_test.go

```diff
@@ -0,0 +1,29 @@
+package create
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+func ExampleSdkConfigBuilder_Build() {
```

Comment:
Is this an example or a test?

Reply:
I was going to add tests later on, this was just the intial PR to get feedback on the use of interfaces for checks. This stuff will be reviewed at least 3 more times after this: from Doc-Scan client -> Doc-Scan, Doc-Scan -> development, and in the release branch.

Reply:
OK, I would put a big fat TODO on it then, it least it's clear something needs to be done before going to prod (not usually fan of TODOs)

Reply:
I am a fan of TODOs in code, but only if they're all removed before release! 😁 which obviously has the potential to sometimes be missed...
Will add

Reply:
Based on https://blog.golang.org/examples, should examples have _test on the end as well?

Reply:
> As with typical tests, examples are functions that reside in a package's _test.go files. Unlike normal test functions, though, example functions take no arguments and begin with the word Example instead of Test.

Reply:
oh! I did not know that. Looks fine then.